src/python/pants/backend/python/subsystems/tailor_python.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from typing import cast
+
+from pants.option.subsystem import Subsystem
+
+
+class TailorPythonSubsystem(Subsystem):
+    options_scope = "tailor-python"
```

Comment:
Rather than Yet Another Subsystem, perhaps `[python-setup].tailor_ignore_solitary_init_files`?

Reply:
Yeah, I like that better, if you think that's an appropriate place for it.